### PR TITLE
Ckan ui#33

### PR DESCRIPTION
**Pull request by jachurchill**

### CODE REVIEW

File: frontend/src/components/Footer.vue [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
       <!-- Navbar content -->
         <v-row>
-            <v-btn text id="footer-home" href="https://www.gov.bc.ca/">{{$tc('Home')}}</v-btn>
-            <v-btn text id="footer-about" href="https://www2.gov.bc.ca/gov/content/about-gov-bc-ca">{{$tc('About gov.bc.ca')}}</v-btn>
-            <v-btn text id="footer-disclaimer" href="http://gov.bc.ca/disclaimer/">{{$tc('Disclaimer')}}</v-btn>
-            <v-btn text id="footer-privacy" href="http://gov.bc.ca/privacy/">{{$tc('Privacy')}}</v-btn>
-            <v-btn text id="footer-accessibility" href="http://gov.bc.ca/webaccessibility/">{{$tc('Accessibility')}}</v-btn>
-            <v-btn text id="footer-copyright" href="http://gov.bc.ca/copyright">{{$tc('Copyright')}}</v-btn>
-            <v-btn text href="https://www2.gov.bc.ca/gov/content/home/contact-us">{{$tc('Contact Us')}}</v-btn>
+            <a class="px-3 pt-2" id="footer-home" href="https://www.gov.bc.ca/">{{$tc('Home')}}</a>
```
Comment (BrandonSharratt) [inline frontend/src/components/Footer.vue]: For consistency this should be v-btns that are depressed with the px mx etc as needed

File: frontend/src/components/Header.vue [hunk 2]
```diff
@@ -33,58 +33,56 @@
                     </v-list>
                 </v-col>
             </v-row>
-          </v-menu>
+          </v-menu> -->
+
+          <v-btn v-if="!loggedIn" depressed text large id="login-btn" class="hidden-sm-and-down" :href="logInUrl" @click="clearStorage" height="100%">{{$tc("LogIn")}}</v-btn>
```
Comment (BrandonSharratt) [inline frontend/src/components/Header.vue]: This should probably be a modification to the User component instead

File: frontend/src/components/pages/home.vue [hunk 3]
```diff
@@ -1,12 +1,20 @@
 <template>
-  <v-container fluid primary align-center justify-center fill-height class="raise">
+  <v-container fluid align-center justify-center fill-height class="raise">
 
         <v-row>
             <v-col cols=12>
-                <p style="font-size: 100pt">BC 💖 {{$tc('Data')}}</p>
+                <v-row align="center" justify="center">
+                    <v-card tile class="mx-3 pa-6" height="200px" width="200px" ><p style="font-size: 50pt">BC</p></v-card>
```
Comment (BrandonSharratt) [inline frontend/src/components/pages/home.vue]: This works for now, might want to consider adding the photo and text areas of the card though based on the design

File: frontend/src/components/pages/home.vue [hunk 4]
```diff
@@ -1,12 +1,20 @@
...
+                    <v-card tile class="mx-3 pa-6" height="200px" width="200px"><p style="font-size: 50pt">💖</p></v-card>
+                    <v-card tile class="mx-3 pa-6" height="200px" width="200px"><p style="font-size: 50pt">{{$tc('Data')}}</p></v-card>
+                </v-row>
             </v-col>
             <v-col cols=12>
-                <v-text-field ref="searchBox" filled background-color="text" append-icon="search" light v-model="searchText" :label="$tc('SearchDatasets')" v-on:keyup="search"></v-text-field>
+                <v-row align="center" justify="center">
+                    <v-col xs="12" md="8">
+                        <v-text-field ref="searchBox" outlined background-color="white" color="primary" append-icon="mdi-magnify" v-model="searchText" :label="$tc('SearchDatasets')" v-on:keyup="search"></v-text-field>
```
Comment (BrandonSharratt) [inline frontend/src/components/pages/home.vue]: Should avoid using color="white" and instead do things like color="primary" or "text" as it's more themeable we aren't currently enabling theme toggling but would require less work in the future.

File: frontend/src/main.js [hunk 5]
```diff
@@ -169,7 +169,7 @@ analyticsServ.ga().then( (gajson) => {
 
   let vuetifyOpts = {
     icons: {
-      iconfont: 'md', // 'md' || 'mdi' || 'fa' || 'fa4'
+      iconfont: 'mdi', // 'md' || 'mdi' || 'fa' || 'fa4'
```
Comment (BrandonSharratt) [inline frontend/src/main.js]: This is a larger scope change and you haven't changed all of the icons across the entire app

Comment (jachurchill) [reply frontend/src/main.js]: addressing this per page as I get to them

File: frontend/src/components/pages/home.vue (hunk 4 above)
Comment (jachurchill) [reply frontend/src/components/pages/home.vue]: this should be good now

File: frontend/src/components/pages/home.vue (hunk 3 above)
Comment (jachurchill) [reply frontend/src/components/pages/home.vue]: I think this is best left for later, we have no idea what's going in the cards yet

File: frontend/src/components/Header.vue (hunk 2 above)
Comment (jachurchill) [reply frontend/src/components/Header.vue]: Was a pain to adjust the styling when it was in there, I don't feel like it's worth dealing with on our current timeline

File: frontend/src/components/Footer.vue (hunk 1 above)
Comment (jachurchill) [reply frontend/src/components/Footer.vue]: done

File: frontend/src/components/pages/home.vue (hunk 4 above)
Comment (BrandonSharratt) [reply frontend/src/components/pages/home.vue]: You're still using white `background-color="white"`

File: frontend/src/components/Header.vue (hunk 2 above)
Comment (BrandonSharratt) [reply frontend/src/components/Header.vue]: It's incurring technical debt to do so as it's more to manage in the 4 versions in the header

File: frontend/src/main.js (hunk 5 above)
Comment (BrandonSharratt) [reply frontend/src/main.js]: I suggest leaving this change out until they are all done., it shouldn't be too hard to do a change on all the icons either though.

File: frontend/src/components/Header.vue (hunk 2 above)
Comment (jachurchill) [reply frontend/src/components/Header.vue]: Agreed but not worth time investment for mvp

File: frontend/src/components/pages/home.vue (hunk 4 above)
Comment (jachurchill) [reply frontend/src/components/pages/home.vue]: resolved

File: frontend/src/main.js (hunk 5 above)
Comment (jachurchill) [reply frontend/src/main.js]: Agreed but is already resolved on branch for next ticket, will be resolved with that one

**Comment by SteveChapmanBCDX** [comment]
Let's get this PR sorted out.  I want to see what the views look like in Toyger.

**Pull request merged**